tests/tools/test_schema_tools.py

```diff
@@ -37,5 +37,5 @@ def test_basic_json_schema(mocker):
     mocked_create_js = mocker.patch(
         "arche.tools.schema.create_json_schema", return_value=schema, autospec=True
     )
```

Comment:
About testing, since these are, basically, print statements I'm not sure if we need to test.
It'd be nice, but if it is too hard, then we can leave as it is.